lnwallet/channel.go

```diff
@@ -4702,6 +4666,16 @@ func (lc *LightningChannel) computeView(view *HtlcView,
 	return ourBalance, theirBalance, totalCommitWeight, filteredHTLCView, nil
 }
 
+func (lc *LightningChannel) recordSettlement(
```

Comment:
nit: missing godoc, hmm I wonder why the linter did not complain

Reply:
linter only complains about godocs for exported functions, I think. I'll include anyway though.